stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/channels/ChannelsViewModel.kt

```diff
@@ -82,6 +82,10 @@ public class ChannelsViewModel(
         }.exhaustive
     }
 
+    public fun leaveChannel(channel: Channel) {
+        chatDomain.useCases.leaveChannel(channel.cid).enqueue()
+    }
```

Comment:
This is lacking some error handling, but not sure if we have any specific behaviour we'd want here on an error

Reply:
For now, channel won't be removed from the list without any error. We also don't handle error in hiding channel method so I would leave it for now, but definitely, we need to think how to handle such cases